.github/workflows/ci_workflows.yml

```diff
@@ -0,0 +1,82 @@
+name: CI
+
+on:
+  push:
+  #  branches:  #uncomment to run only on master
+  #  - master
+  #  tags: # run CI if specific tags are pushed
+  pull_request:
+  #  branches: # only build on PRs against 'main' if you need to further limit when CI is run.
+  #  - main
+
+jobs:
+  # Github Actions supports ubuntu, windows, and macos virtual environments:
+  # https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners
+  ci_tests:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.allow_failure}}
+    strategy:
+      matrix:
+        include:
+          - name: Code style checks
+            os: ubuntu-latest
+            python: 3.x
+            toxenv: codestyle
+            allow_failure: false
+
+          - name: PEP 517
+            os: ubuntu-latest
+            python: 3.x
+            toxenv: pep517
+            allow_failure: true
+
+          - name: Python 3.8 with coverage checking
+            os: ubuntu-latest
+            python: 3.8
+            toxenv: py38-test-cov
+            allow_failure: false
+
+          - name: OS X - Python 3.8
+            os: macos-latest
+            python: 3.8
+            toxenv: py38-test
+            allow_failure: false
+
+          - name: Windows - Python 3.8
+            os: windows-latest
+            python: 3.8
+            toxenv: py38-test
+            allow_failure: false
+
+          - name: Documentation build
+            os: ubuntu-latest
+            python: 3.8
+            toxenv: build_docs
+            allow_failure: false
+
+    env:
+      SETUP_XVFB: True
```

Comment:
Do you still need this? Someone said this is leftover from `astropy-helpers`.

Reply:
Good question.  I assumed it was necessary to get the UI stuff to pop up but lemme try removing at to see what happens

Reply:
Ah, thanks @pllim - based on https://github.com/eteq/jdaviz/actions/runs/373116419 it looks like indeed it is unnecessary